pkg/sql/logic_test.go

```diff
@@ -819,6 +826,7 @@ func (t *logicTest) processTestFile(path string) error {
 				//   - I for integer
```

Comment:
`T` has become somewhat of a dumping ground for all other types (arrays, timestamps, etc.). Since you're here, we might as well document this.